test/Cli/Func.E2ETests/func-start.Tests/InProcTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Lets use camelCase for the folder structure. we only want func-start in the src/Cli/func

i.e.

`test/Cli/Func.E2ETests/Commands/FuncStart/InProcTests.cs`